engine/app/views/shared/_jobs_table.erb

```diff
@@ -19,7 +19,10 @@
             <td><%= job.queue_name %></td>
             <td><%= job.scheduled_at || job.created_at %></td>
             <td><%= job.error %></td>
-            <td><pre><%= JSON.pretty_generate(job.serialized_params) %></pre></td>
+            <td>
+              <%= link_to "Preview", "#params-#{job.id}", "role" => "button", "data-bs-toggle" => "collapse", "aria-expanded" => false, "aria-controls" => "#params-#{job.id}" %>
+              <pre class="collapse" id="params-<%= job.id %>"><%= JSON.pretty_generate(job.serialized_params) %></pre>
```

Comment:
It's more semantic/accessible to use a button rather than link for the toggle. I also suggest using the `dom_id(job, "params")` helper to construct the target.  

```suggestion
              <% collapse_target = "#{dom_id(job)}-params" %>
              <%= button_tag "Toggle Params", class: "btn btn-link", type: "button", "data-bs-toggle": "collapse", "data-bs-target": "##{collapse_target}", "aria-expanded": false, "aria-controls": "##{collapse_target}" %>
              <pre class="collapse" id="<%= collapse_target %>"><%= JSON.pretty_generate(job.serialized_params) %></pre>
```

Reply:
👍